### ISSUE TITLE
spaCy tools do not appear in the logs

### ISSUE THREAD

**Issue by koaning** [RasaHQ/rasa#10814]
In Rasa 3.0, when I run this pipeline:

 pipeline:

*   name: SpacyNLP
     model: en_core_web_md
*   name: SpacyTokenizer
*   name: SpacyFeaturizer
*   name: RegexFeaturizer
*   name: LexicalSyntacticFeaturizer
*   name: CountVectorsFeaturizer
*   name: CountVectorsFeaturizer
     analyzer: char_wb
     min_ngram: 1
     max_ngram: 4
*   name: DIETClassifier
     epochs: 100
*   name: EntitySynonymMapper
*   name: ResponseSelector
     epochs: 100
*   name: FallbackClassifier
     threshold: 0.3
     ambiguity_threshold: 0.1

This is the output during training:

 The configuration for policies was chosen automatically. It was written into the config file at 'config.yml'.
 2022-02-04 15:41:24 INFO rasa.nlu.utils.spacy_utils - Trying to load SpaCy model with name 'en_core_web_md'.
 2022-02-04 15:41:26 INFO rasa.nlu.utils.spacy_utils - Trying to load SpaCy model with name 'en_core_web_md'.
 2022-02-04 15:41:28 INFO rasa.engine.training.hooks - Starting to train component 'RegexFeaturizer'.
 2022-02-04 15:41:28 INFO rasa.engine.training.hooks - Finished training component 'RegexFeaturizer'.
 2022-02-04 15:41:28 INFO rasa.engine.training.hooks - Starting to train component 'LexicalSyntacticFeaturizer'.
 2022-02-04 15:41:29 INFO rasa.engine.training.hooks - Finished training component 'LexicalSyntacticFeaturizer'.
 2022-02-04 15:41:29 INFO rasa.engine.training.hooks - Starting to train component 'CountVectorsFeaturizer'.
 2022-02-04 15:41:29 INFO rasa.nlu.featurizers.sparse_featurizer.count_vectors_featurizer - 74 vocabulary items were created for text attribute.
 2022-02-04 15:41:29 INFO rasa.engine.training.hooks - Finished training component 'CountVectorsFeaturizer'.
 2022-02-04 15:41:29 INFO rasa.engine.training.hooks - Starting to train component 'CountVectorsFeaturizer'.
 2022-02-04 15:41:29 INFO rasa.nlu.featurizers.sparse_featurizer.count_vectors_featurizer - 683 vocabulary items were created for text attribute.
 2022-02-04 15:41:29 INFO rasa.engine.training.hooks - Finished training component 'CountVectorsFeaturizer'.
 2022-02-04 15:41:29 INFO rasa.engine.training.hooks - Starting to train component 'DIETClassifier'.
 /home/gitpod/.pyenv/versions/3.8.12/lib/python3.8/site-packages/rasa/utils/train_utils.py:527: UserWarning: constrain_similarities is set to `False`. It is recommended to set it to `True` when using cross-entropy loss.
 rasa.shared.utils.io.raise_warning(
 Epochs: 100%|████████████████████████████████████████████████████████████████████████████████| 100/100 <span class="error">[00:24&lt;00:00, 4.10it/s, t_loss=0.968, i_acc=1]</span>
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Finished training component 'DIETClassifier'.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Starting to train component 'EntitySynonymMapper'.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Finished training component 'EntitySynonymMapper'.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Starting to train component 'ResponseSelector'.
 2022-02-04 15:41:54 INFO rasa.nlu.selectors.response_selector - Retrieval intent parameter was left to its default value. This response selector will be trained on training examples combining all retrieval intents.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Finished training component 'ResponseSelector'.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Restored component 'MemoizationPolicy' from cache.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Restored component 'RulePolicy' from cache.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Restored component 'TEDPolicy' from cache.
 2022-02-04 15:41:54 INFO rasa.engine.training.hooks - Restored component 'UnexpecTEDIntentPolicy' from cache.
 Your Rasa model is trained and saved at 'models/20220204-154122-vertical-font.tar.gz'.

It seems that the spaCy components aren't logged. It makes a bit of sense because technically these are not "training" but it seems preferable to have the components log that they are at least ready. Without such a log, it confused me because I was wondering if I had misconfigured something.
